### PR TITLE
Localscriptcreator

### PR DESCRIPTION
**Pull request by hlasimpk**
Added a local script creator class and test function. 

Would like advice on the doc strings as we're inputting a function and I don't know how to tag it. I'm also just returning the collector object from a `LocalScriptCreator.collector()` so you might have some suggestions about how to rewrite this in a nicer way.

### CODE REVIEW

**Comment by fsimkovic** [comment]
@hlasimpk test cases don't pass

**Comment by hlasimpk** [comment]
@fsimkovic my pytest install seems to be missing attributes, any idea?

File: pyjob/tests/test_script.py [hunk 1]
```diff
@@ -247,3 +248,18 @@ def test_3(self):
...
+        for c in cmd:
+             script.append(' '.join(map(str, c)))
+        return script
+
+    def test_1(self):
+        nproc = 2
+        options = [1, 2, 3, 4, 5]
+        script_creator = LocalScriptCreator(func=self.example_function, iterable=options, processes=nproc)
+        collector = script.creator.collector()
+        assert collector.scripts == [['e c h o   1'], ['e c h o   2'], ['e c h o   3'], ['e c h o   4'], ['e c h o   5']]
```
Comment (fsimkovic) [inline pyjob/tests/test_script.py]: What's with the spaces in all `str` objects?

File: pyjob/tests/test_script.py [hunk 2]
```diff
@@ -247,3 +248,18 @@ def test_3(self):
         else:
             assert ScriptProperty.SHELL.shebang == '#!/bin/bash'
             assert ScriptProperty.SHELL.suffix == '.sh'
+
+class TestLocalScriptCreator(object):
+    def example_function(self, i):
```
Comment (fsimkovic) [inline pyjob/tests/test_script.py]: Make this a `@staticmethod`.

**Comment by fsimkovic** [comment]
Please add an example to the documentation. This should be documented in `docs/quickstart.rst`.

**Review by fsimkovic — CHANGES_REQUESTED** [review]

**Comment by hlasimpk** [comment]
Sorted out the test cases and added description to docs

**Comment by fsimkovic** [comment]
@hlasimpk Please add a `@pytest.skipif` decorator around the `LocalScriptCreator` class on Windows ... check the AppVeyor report above

**Review by fsimkovic — APPROVED** [review]

**Pull request merged**